library/Ivoz/Kam/Infrastructure/Gearman/TrunksClient.php

```diff
@@ -107,7 +107,8 @@ public function getLcrGatewayInfo($gw_id): array
     {
         $response = $this->sendRequest(
             self::LCR_DUMP_GWS_ACTION,
-            [$gw_id]
+            [$gw_id],
+            1
```

Comment:
I have reconsidered this and I prefer 2 seconds here, for a max 50 sec in a klear list screen